src/rpc/client.cpp

```diff
@@ -155,6 +155,7 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "getblockstats", 0, "hash_or_height" },
     { "getblockstats", 1, "stats" },
     { "pruneblockchain", 0, "height" },
+    { "dumpcoinstats", 1, "cummulative"},
```

Comment:
Missed a spot

Reply:
Gah, thought I had grepped for it. But there was another silent merge conflict anyway... :/